package.json

```diff
@@ -118,5 +118,19 @@
     "help": "yamscripts help",
     "yamscripts": "yamscripts compile",
     "__": "# NOTE: THESE SCRIPTS ARE COMPILED!!! EDIT yamscripts.yml instead!!!"
-  }
+  },
+  "preferences": [
+    {
+      "name": "performanceLog",
```

Comment:
I'd say to change it to `performance.log` to remain in style with what we do in gecko. We will need to change it when we do land there anyway.